### ISSUE TITLE
Bug: Large font sizes cause menu to break

### ISSUE THREAD

**Issue by fancyham** [hackforla/food-oasis#2508]
#### Describe the bug 

Large font sizes cause menu to break and go off-screen

#### Steps to reproduce the issue

1. on an iPhone
2. Go to foodoasis.la in Safari.  Click on the hamburger menu
3. Change font zoom size to really large

![Image](https://github.com/user-attachments/assets/de8f8c1e-ed51-46f2-b523-e418c2ef821a)

#### What's the expected result?

- Menu should be readable, word wrap

(if you have a suggested solution, you can add it here)

**Comment by fancyham** [comment]
https://github.com/user-attachments/assets/515263ac-5c19-4f29-9c3d-fe3d2126a15f

In Safari, tap the 'page' menu in the address bar and increase the font size using the bottom buttons
